applications/nrf5340_audio/src/audio/Kconfig

```diff
@@ -45,6 +45,12 @@ config AUDIO_MAX_PRES_DLY_US
 
 choice AUDIO_SYSTEM_SAMPLE_RATE
 	prompt "System audio sample rate"
```

Comment:
We should adjust the prompts to make it more clear what is the HW and what is "over the air"

Reply:
Changed prompt for BT_AUDIO_SAMPLE_RATE to include BT